client/src/component/ticketForm.js

```diff
@@ -0,0 +1,82 @@
+import React, { useState } from 'react'
+import Button from '@material-ui/core/Button';
+import TextField from '@material-ui/core/TextField';
+import Grid from '@material-ui/core/Grid';
+import axios from 'axios';
+
+
+const TicketForm = () => {
+    //useState
+    const [firstName, setFirstName] = useState('')
+    const [lastName, setLastName] = useState('')
+    const [phone, setPhone] = useState('')
+    const [pickUpDate, setPickUpDate] = useState('')
+    const [ticketNumber, setTicketNumber] = useState('')
+    const [dropDate, setDropDate] = useState('')
+    // const handleChange = (event) => {
+    //     setFirstName(event.target.value);
+    //     console.log(event.target);
+    //   };
+
+    //Axios Put request 
```

Comment:
```suggestion
```

Generally, comments are there to provide context surround what the code is doing. We can see that the code is going to make a PUT w/ Axios, so comments like these aren't really necessary.